### ISSUE TITLE
[Feature request] Mappings for go next/previous

### ISSUE THREAD

**Issue by dkaszews** [mengelbrecht/lightline-bufferline#114]
Hi! Just a small request: there are already mappings for `go(n)`, but often I want to go to the next or previous one without looking at whichever I have open. This is similar to `bn` and `bN`, but those may open buffers that are filtered out by bufferline.

I wrote a function for it, but thought other people may appreciate it if it was integrated into the plugin itself. Feel free to copy the code, or let me know if you'd like me to create the pull request.

```vim
function LightlineBufferLineGoNext(...)
    let offset = get(a:, 1, 1)

    let index = bufnr('%')
    let ordinal = lightline#bufferline#get_ordinal_number_for_buffer(index)
    if ordinal < 0
        echoerr 'Buffer not on tabline'
        return
    endif

    let buffers = lightline#bufferline#buffers()
    let count = len(buffers[0]) + len(buffers[1]) + len(buffers[2])

    let ordinal += offset
    if ordinal < 1
        let ordinal = count
    elseif ordinal > count
        let ordinal = 1
    endif

     let index = lightline#bufferline#get_buffer_for_ordinal_number(ordinal)
     if index < 0
         echoerr 'Buffer not found: ' .. ordinal
         return
     endif
     execute 'b' .. index
 endfunction

map <Tab> :call LightlineBufferLineGoNext()<CR>
map <S-Tab> :call LightlineBufferLineGoNext(-1)<CR>
```

**Comment by mengelbrecht** [comment]
Hi @dkaszews thanks for the suggestion. I added the feature in #115. Please let me know if it works for you.

**Comment by mengelbrecht** [comment]
@dkaszews this feature has been merged (e01bde9c58b3431dd9ef03a11bc7e022c9f8f060)
